lib/runner/errors/constants.js

```diff
@@ -0,0 +1,34 @@
+const INVALID_LOOKUP_STRATEGY_ERROR = 'runtime~extractRunnableItems: Invalid entrypoint lookupStrategy',
+    INVALID_ENTRY_POINTS_ERROR = 'runtime~extractRunnableItems: Invalid entrypoints',
+    // eslint-disable-next-line no-template-curly-in-string
+    INVALID_FOLDER_OR_REQUEST = 'Unable to find a folder or request: ${folderOrRequestName}',
+    errors = {
+        invalidLookupStrategy: 'invalidLookupStrategy',
+        invalidEntryPoints: 'invalidEntryPoints',
+        invalidFolderOrRequest: 'invalidFolderOrRequest'
+    },
+    errorMessages = {
+        invalidLookupStrategy: INVALID_LOOKUP_STRATEGY_ERROR,
+        invalidEntryPoints: INVALID_ENTRY_POINTS_ERROR,
+        invalidFolderOrRequest: INVALID_FOLDER_OR_REQUEST
+    },
+    errorCodes = {
+        genericError: -1,
+        invalidLookupStrategy: 1,
+        invalidEntryPoints: 2,
+        invalidFolderOrRequest: 3
+    },
+    messageGenerators = {
+        // eslint-disable-next-line no-unused-vars
+        invalidFolderOrRequest: ({ folderOrRequestName }) => {
+            // eslint-disable-next-line no-template-curly-in-string
+            return INVALID_FOLDER_OR_REQUEST.replace('${folderOrRequestName}', folderOrRequestName);
```

Comment:
Should we shift message generation into respective error class?
That way we won't need template variables